### ISSUE TITLE
Runtime error in styled components

### ISSUE THREAD

**Issue by JackSkylark** [technekes/cast-ui#138]
styled-components is throwing a `StyledComponentsError` exception when using cast-ui. I believe that this is due to the `Modal.component.tsx` component incorrectly importing ReactModal and then trying to extend styles. If I'm reading the types right, this should not be a namespace import.

The stack trace that I'm getting shows that the error is being thrown in the `constructWithOptions` function and the `tag` being provided is an object of the structure:
```
{
    default: function,
    __moduleExports: function 
}
```

I'll also be looking into to make sure that this isn't an issue with my babel setup.

**Comment by ben-dalton** [comment]
Thank you, @JackSkylark for raising this. We were looking into it yesterday and I believe it has to do with the build process. I've pushed a new version that seems to avoid the styled-components error, but it requires you to import as below: 
```
import { Alert } from '@tkxs/cast-ui/lib-esm';
```

Will update the readme.

**Comment by JackSkylark** [comment]
Hey @ben-dalton! Thank you for taking a look into this issue. I've updated the version of cast-ui to `0.1.20` but I'm still seeing the same issue. 

Following up on my investigation earlier, I attempted to use the dependency of `react-modal` outside of cast-ui and I do indeed to need to import from the default export rather than importing all of the module exports. (aka `import ReactModal from "react-modal"` instead of `import * as ReactModal from "react-modal` if I'm expecting to import the React component)
